releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -0,0 +1,17 @@
+## Microsoft Extensions Azure App Configuration
+
+The Azure App Configuration Extension in the Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview).
+
+
+### v1.0.0-preview-008920001-990
+
+* Added `TrimKeyPrefix` method to remove prefixes from the list of returned key-values from the config store.
+* Improved durability on Watch, so that transient network errors do not cause the process to detach.
+* Retries on network error when listing keys in Managed Identity Connector.
```

Comment:
>Retries on network error when listing keys in Managed Identity Connector [](start = 2, length = 72)

Should read `Added additional retries on network error when using `ConnectWithManagedIdentity`.